src/Build/BackEnd/Components/ProjectCache/ProjectCacheService.cs

```diff
@@ -413,8 +420,14 @@ public async Task ShutDown()
             {
                 SetState(ProjectCacheServiceState.ShutdownStarted);
 
+                logger.LogMessage("Shutting down project cache plugin", MessageImportance.Low);
+                var timer = Stopwatch.StartNew();
+
                 await _projectCachePlugin.EndBuildAsync(logger, _cancellationToken);
 
+                timer.Stop();
+                logger.LogMessage($"Finished shutting down project cache plugin in {timer.Elapsed.TotalMilliseconds} ms", MessageImportance.Low);
```

Comment:
Should these be localized? Or is this logger just for dev purposes?

Reply:
Eventually, yes, when we move the project cache out of the experimental namespace. But for now it's only internal users seeing it, on debug verbosity. And at some point I want to redo the logging side of the plugin so these strings might even go away.